.github/workflows/typos.yml

```diff
@@ -0,0 +1,24 @@
+# Spell checker
+
+name: Typos Action
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  run:
+    name: Spell Check with Typos
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v45.0.2
```

Comment:
This dependency has come under Occam's razor because https://www.stepsecurity.io/blog/harden-runner-detection-tj-actions-changed-files-action-is-compromised

This requires more effort during the implementation phase.  However, once this is done, we will not need to get the list of changed files on future requests, because the rest of the documentation will already be validated.